src/main/scala/mesosphere/marathon/core/task/tracker/impl/TaskLoader.scala

```diff
@@ -0,0 +1,9 @@
+package mesosphere.marathon.core.task.tracker.impl
+import scala.concurrent.Future
+
+/**
+  * Loads all task data into an [[AppDataMap]].
+  */
+private[tracker] trait TaskLoader {
```

Comment:
Not important: since this is a simple helper class, I wonder if we need a trait for that.